src/extension/log/LogHelper.ts

```diff
@@ -58,6 +59,15 @@ export function getLoggingDirectory(): string | null {
     return null;
 }
 
+export function isVerboseLogLevel(logLevel: string | undefined): boolean {
```

Comment:
Why string?

Reply:
`isVerboseLogLevel` function was removed. Now LogLevel for CDP Proxy is determined [here](https://github.com/microsoft/vscode-react-native/blob/c3400dc7306165bfcf73af591c9c096f1a1894a9/src/debugger/rnDebugSession.ts#L191-L198).